pageserver/src/reltag.rs

```diff
@@ -75,6 +75,35 @@ impl fmt::Display for RelTag {
     }
 }
 
+impl RelTag {
+    /// Formats:
+    /// <oid>
+    /// <oid>_<fork name>
+    /// <oid>.<segment number>
+    /// <oid>_<fork name>.<segment number>
```

Comment:
This is now outdated, and if we update it it will be more complex than the code itself :)
```suggestion
```